custom_components/tesla_custom/sensor.py

```diff
@@ -175,7 +175,7 @@ def __init__(
         self._attr_native_unit_of_measurement = POWER_KILO_WATT
 
     @property
-    def native_value(self) -> int:
+    def native_value(self) -> float:
```

Comment:
This a typing hint and actually won't change anything. Did you test to confirm it actually does anything?